container-images/candlepin/Containerfile

```diff
@@ -2,8 +2,13 @@ FROM quay.io/centos/centos:stream9
 RUN dnf -y update && \
     dnf clean all
 RUN dnf -y --nodocs install \
+    gettext \
```

Comment:
This provides `envsubst`. On Fedora 41 it's split off into `gettext-envsubst` so I think it'll be the same on EL10.